example/eventGenerator/EventTelemetryProvider.js

```diff
@@ -33,7 +33,7 @@ class EventTelemetryProvider {
 
     generateData(firstObservedTime, count, startTime, duration, name) {
         const millisecondsSinceStart = startTime - firstObservedTime;
-        const utc = Math.floor(startTime / duration) * duration;
+        const utc = startTime + (count * duration);
```

Comment:
allows any request to show data. previous incarnation was showing data well into the future